From 3c25c398a02485e892abde216fbb905808e4151a Mon Sep 17 00:00:00 2001
From: Dan Elphick <delphick@chromium.org>
Date: Tue, 22 Jun 2021 22:47:41 +0100
Subject: [PATCH] Fix riscv64 build due to base/strings.h change

Bug: v8:11917
Change-Id: Ie80a24bc9826aa42d7a35dcebda3bd4ed5be05ba
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2980971
Auto-Submit: Dan Elphick <delphick@chromium.org>
Commit-Queue: Ji Qiu <qiuji@iscas.ac.cn>
Reviewed-by: Ji Qiu <qiuji@iscas.ac.cn>
Cr-Commit-Position: refs/heads/master@{#75314}
---
 src/diagnostics/riscv64/disasm-riscv64.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/diagnostics/riscv64/disasm-riscv64.cc b/src/diagnostics/riscv64/disasm-riscv64.cc
index 043c33f62f..2955612166 100644
--- a/src/diagnostics/riscv64/disasm-riscv64.cc
+++ b/src/diagnostics/riscv64/disasm-riscv64.cc
@@ -1864,7 +1864,7 @@ int Decoder::InstructionDecode(byte* instr_ptr) {
 namespace disasm {
 
 const char* NameConverter::NameOfAddress(byte* addr) const {
-  v8::internal::base::SNPrintF(tmp_buffer_, "%p", static_cast<void*>(addr));
+  v8::base::SNPrintF(tmp_buffer_, "%p", static_cast<void*>(addr));
   return tmp_buffer_.begin();
 }
 
-- 
2.35.1

